cpp/ql/src/Best Practices/Likely Errors/CommaBeforeMisleadingIndentation.ql

```diff
@@ -0,0 +1,53 @@
+/**
+ * @name Comma before misleading indentation
+ * @description The expressions before and after the comma operator can be misread because of an unusual difference in start columns.
```

Comment:
We usually aim to structure the description as "syntax X causes problem Y". It's more difficult with non-security queries. Would this work?

```suggestion
 * @description If expressions before and after a comma operator use different indentation, it is easy to misread the purpose of the code.
```